README.md

```diff
@@ -25,3 +25,53 @@ You need to do a few things to contribute to the documentation:
 ## Styling the Docs
 
 If you want to change the look and feel of the ownCloud documentation, you can find all the information that you need in [the docs-ui repository](https://github.com/owncloud/docs-ui/blob/master/README.adoc).
+
+## How To Generate PDFs
+
+To generate a PDF version of one or more of the manuals, you need to have `asciidoctor-pdf` installed.
+With it installed, run the command below, to generate a PDF version of a manual.
+Please be aware that, depending on the size of the manual, PDF generation may take some time.
+
+```console
+asciidoctor-pdf \
+    book.adoc \
+    -a pdf-stylesdir=resources/themes \
+    -a pdf-style=owncloud
+```
```

Comment:
From where in the directory do you start that command and what/where/name will the output be?

Reply:
Good point. Thanks for spotting that omission.